.github/workflows/schemacode_ci.yml

```
@@ -51,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ["ubuntu-latest"]
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
         python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Given that it's all filesystem manipulation and not weird numerical edge cases, it seems like overkill to test the whole Python range on all three platforms. I would suggest:

```suggestion
        os: ["ubuntu-latest"]
        python-version: ["3.7", "3.8", "3.9", "3.10"]
        include:
          - os: macos-latest
            python-version: 3
          - os: windows-latest
            python-version: 3
```